src/EventListener/FrontendListener.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of terminal42/contao-inserttags.
+ *
+ * (c) terminal42
+ *
+ * @license MIT
+ */
+
+namespace Terminal42\InsertTagsBundle\EventListener;
+
+use Contao\CoreBundle\InsertTag\InsertTagParser;
+use Contao\CoreBundle\Monolog\ContaoContext;
+use Contao\CoreBundle\ServiceAnnotation\Hook;
+use Contao\CoreBundle\String\SimpleTokenParser;
+use Doctrine\DBAL\Connection;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Terminal42\InsertTagsBundle\InsertTagHandler;
+
+class FrontendListener
+{
+    private Connection $connection;
+    private InsertTagHandler $handler;
+    private InsertTagParser $insertTagParser;
+    private LoggerInterface $logger;
+    private SimpleTokenParser $simpleTokenParser;
+
+    private array $internalRegistry = [];
+
+    public function __construct(Connection $connection, InsertTagHandler $handler, InsertTagParser $insertTagParser, LoggerInterface $logger, SimpleTokenParser $simpleTokenParser)
+    {
+        $this->connection = $connection;
+        $this->handler = $handler;
+        $this->insertTagParser = $insertTagParser;
+        $this->logger = $logger;
+        $this->simpleTokenParser = $simpleTokenParser;
+    }
+
+    /**
+     * @Hook("outputFrontendTemplate")
+     */
+    public function onOutputFrontendTemplate(string $buffer): string
```

Comment:
This has to be removed.

Reply:
Completely? I wondered why it was even in the module source code…

Reply:
Yeah because this cache stuff is not supported anymore.